WFD/swb_import.py

```diff
@@ -213,7 +216,9 @@ def make_significant_impact_type(self, data):
                * In both cases the reference can still be added.
 
         :param data: dict of data for a single swb
+        :return: [Statement]
         """
+        # @todo: Support for UNKN, OTHE (somevalue) + NOTA, NOSI (novalue)
```

Comment:
If I understand the docstring correctly, if there's no impact type in the data, it means it's been _determined_ there are none? Just trying to wrap my head around the use of 'novalue' here.